generated/nidaqmx/system/device.py

```diff
@@ -130,6 +131,52 @@ def do_ports(self):
 
     # endregion
 
+    # region Calibration Info property
+
+    @property
+    def ext_cal_last_date_and_time(self):
+        """
+        Indicates the last date and time that the device underwent an
+        external calibration.
```

Comment:
PEP-257 says the 1st line of the docstring is a summary. Please run Sphinx and check the output in a web browser to make sure this line wrapping is ok.

Reply:
CONTRIBUTING.md gives details on how to build the documentation

Reply:
@zhindes Thanks for pointing how to build the documentation.

After building the documentation, the 1st line of docstring does not wrap in maximum size of browser. And the docstring is wrapped in two lines if browser size is set to half.

browser max size:
![image](https://github.com/ni/nidaqmx-python/assets/27721199/78f50c9b-8b7d-45a8-9d93-27045cbf2100)

browser half size:
![image](https://github.com/ni/nidaqmx-python/assets/27721199/8eecbc82-b402-4015-b889-eabe202d1971)